KIA_STINGER_HML.xml

```diff
@@ -0,0 +1,10 @@
+<car>
+   <brand>KIA</brand>
+   <model>Stinger</model>
+   <year>2019</year>
+   <color>Yellow</color>
+   <engineCapacity>3.3</engineCapacity>
+   <power>370</power>
+   <issedan>true</issedan>
```

Comment:
If we're using `lowerCamelCase` for naming properties -> then we should use it for all of them

`isSedan`, `isNew` :)